apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -585,13 +585,25 @@ export function buildMachine(
               NO_PAPER_ANYWHERE: 'resetting_state_machine_after_success',
             },
           },
-          // Ballot invalidation is a 2-stage process so the frontend can prompt the voter to get a pollworker
           waiting_for_invalidated_ballot_confirmation: {
-            on: {
-              VOTER_CONFIRMED_INVALIDATED_BALLOT: 'eject_to_front',
-              // Even if ballot is removed from front, we still want the frontend to require pollworker auth before continuing
-              NO_PAPER_ANYWHERE: undefined,
+            initial: 'paper_present',
+            states: {
+              paper_present: {
+                invoke: pollPaperStatus(),
+                on: {
+                  NO_PAPER_ANYWHERE: 'paper_nowhere',
+                },
+              },
+              paper_nowhere: {
+                on: {
+                  POLLWORKER_CONFIRMED_INVALIDATED_BALLOT: 'done',
+                },
+              },
```

Comment:
The copy changes based on whether paper is present

Reply:
paper_nowhere is a little confusing. Maybe no_paper or paper_absent?